cmd/serverless-init/main.go

```diff
@@ -8,108 +8,122 @@
 package main
 
 import (
+	"context"
+	serverlessInitLog "github.com/DataDog/datadog-agent/cmd/serverless-init/log"
+	"github.com/DataDog/datadog-agent/cmd/serverless-init/mode"
+	"github.com/DataDog/datadog-agent/comp/core"
+	"github.com/DataDog/datadog-agent/comp/core/autodiscovery"
+	"github.com/DataDog/datadog-agent/comp/core/autodiscovery/autodiscoveryimpl"
```

Comment:
Do we need this component in the serverless binary?

Reply:
replying in the thread above for autodiscovery+workloadmeta